Formula/odin.rb

```diff
@@ -0,0 +1,37 @@
+class Odin < Formula
+  desc "The Odin Programming Language"
+  homepage "https://odin-lang.org/"
+  url "https://github.com/odin-lang/Odin/archive/v0.12.0.tar.gz"
+  sha256 "8356124c8cc7e08ac39872e5bb10593a412e67f81df621124097facd9b2b26cc"
+  head "https://github.com/odin-lang/Odin.git"
+
+  depends_on "llvm"
+
+  uses_from_macos "iconv"
+
+  def install
+    system "make", "release"
+    libexec.install "odin", "core", "shared"
+    (bin/"odin").write <<~EOS
+      #!/bin/sh
+      export PATH="#{Formula["llvm"].opt_bin}:$PATH"
+      #{libexec}/odin "$@"
+    EOS
```

Comment:
```suggestion
    (bin/"odin").write_env_script(libexec/"odin", :PATH => "#{Formula["llvm"].opt_bin}:$PATH")
```

Reply:
Thanks for the suggestion but at the same time I realized I should use `exec -a` to adjust Odin's `argv0`.